### DISCUSSION TITLE
Plug in packaging

### DISCUSSION THREAD

**Discussion by Chengminshu** [grafana/grafana#43518] [Plugins]
sir,How can I package a plug-in of grafana and introduce it to other projects

**Comment by mckn** [comment]
Can you please explain a bit more what your are trying to do?

**Reply by Chengminshu** [reply]
How can I pull out the time series module of grafana and package it into a package, which can be used in other projects by passing parameters

**Reply by mckn** [reply]
Do you want to reuse it in a panel or app plugin for Grafana? Or in a totally different project?

**Reply by Chengminshu** [reply]
in a totally different project.I want to pull out timeseries and package them for use in any other project.Can you give me some ideas or methods

**Reply by mckn** [reply]
Ah, that is not possible. You can not use it outside of the Grafana context. There are lots of other packages you can use.

**Reply by Chengminshu** [reply]
Do you mean that individual panel components cannot be disassembled and packaged

**Reply by mckn** [reply]
Yes, and the visualizations in `@grafana/ui` still requires the context of Grafana to be able to function properly.

**Reply by Chengminshu** [reply]
Ok，thanks.I try to other's methods

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
